hw/hw2/task4.js

```diff
@@ -0,0 +1,63 @@
+let number = 3;
+
+function createMagicSquare(number){
+    let array =  Array(number);
+    let magicSquare = Array.from(array,(x) => {return 0});
+    magicSquare = magicSquare.map((x) => {return x = Array.from(Array(number),(x) => {return 0})});
```

Comment:
- Don't use comma, it's hard to read 
- Don't use return and assignment in one line 

- (x) => {return 0} should be x => 0

Reply:
fixed